service/src/main/java/uk/nhs/adaptors/gp2gp/gpc/GetGpcStructuredTaskExecutor.java

```diff
@@ -128,14 +129,15 @@ public void execute(GetGpcStructuredTaskDefinition structuredTaskDefinition) {
             messageContext.resetMessageContext();
         }
 
-        var allExternalAttachments = Stream
-            .concat(externalAttachments.stream(), absentAttachments.stream())
-            .collect(Collectors.toList());
+        var allExternalAttachments = Stream.concat(externalAttachments.stream(), absentAttachments.stream())
+            .peek(this::replaceFilenameWithPlaceholder)
+            .peek(this::addPrefixToDocumentId)
+            .toList();
```

Comment:
We also need to be updating the `content_type` field within the JSON ExternalMessage representation, it should be `text/plain` for an AbsentAttachment.

```json
{
    "description": "Filename=\"CC13A057-11A5-435A-913C-731A1670DD88.doc\" ContentType=application/msword Compressed=No LargeAttachment=No OriginalBase64=No Length=236",
    "document_id": "_CC13A057-11A5-435A-913C-731A1670DD88",
    "message_id": "9723E7E1-9FF9-479C-A038-5E0343D485C5"
}
```